app/src/main/java/com/globant/tutoria/subscriberssearch/dataAdapter/SubscribersAdapter.java

```diff
@@ -16,17 +19,23 @@
  */
 public class SubscribersAdapter extends RecyclerView.Adapter<SubscribersAdapter.ViewHolder> {
 
+    private static final int WIDTH_IMG_AVATAR = 80;
+    private static final int HEIGHT_IMG_AVATAR = 80;
```

Comment:
you count get this attributes from the context since you already created some attributes in the dimens.xml